x-pack/test/functional_with_es_ssl/apps/triggers_actions_ui/alerts.ts

```diff
@@ -123,6 +127,54 @@ export default ({ getPageObjects, getService }: FtrProviderContext) => {
       await deleteAlerts(alertsToDelete.map((alertItem: { id: string }) => alertItem.id));
     });
 
+    it('should show save confirmation before creating alert with no actions', async () => {
+      const alertName = generateUniqueKey();
+      await defineAlert(alertName);
+
+      await testSubjects.click('saveAlertButton');
+      await testSubjects.existOrFail('confirmAlertSaveModal');
+      await testSubjects.click('confirmAlertSaveModal > confirmModalConfirmButton');
+      await testSubjects.missingOrFail('confirmAlertSaveModal');
+
+      const toastTitle = await pageObjects.common.closeToast();
+      expect(toastTitle).to.eql(`Saved '${alertName}'`);
+      await pageObjects.triggersActionsUI.searchAlerts(alertName);
+      const searchResultsAfterSave = await pageObjects.triggersActionsUI.getAlertsList();
+      expect(searchResultsAfterSave).to.eql([
+        {
+          name: alertName,
+          tagsText: '',
+          alertType: 'Index threshold',
+          interval: '1m',
+        },
+      ]);
+
+      // clean up created alert
+      const alertsToDelete = await getAlertsByName(alertName);
+      await deleteAlerts(alertsToDelete.map((alertItem: { id: string }) => alertItem.id));
+    });
+
+    it('should not save if save confirmation is cancelled', async () => {
+      const alertsListBefore = await pageObjects.triggersActionsUI.getAlertsList();
+
+      const alertName = generateUniqueKey();
+      await defineAlert(alertName);
+
+      await testSubjects.click('saveAlertButton');
+      await testSubjects.existOrFail('confirmAlertSaveModal');
+      await testSubjects.click('confirmAlertSaveModal > confirmModalCancelButton');
+      await testSubjects.missingOrFail('confirmAlertSaveModal');
+
+      await testSubjects.click('cancelSaveAlertButton');
+
+      const alertsListAfter = await pageObjects.triggersActionsUI.getAlertsList();
+      expect(alertsListBefore.length).to.eql(alertsListAfter.length);
+
+      // clean up created alert
+      const alertsToDelete = await getAlertsByName(alertName);
+      await deleteAlerts(alertsToDelete.map((alertItem: { id: string }) => alertItem.id));
```

Comment:
I don't think that we created the alert in the code above, so no need to clean up.